extensions/parquet/compression/src/main/java/io/deephaven/parquet/compress/LZ4WithLZ4RawBackupCompressorAdapter.java

```diff
@@ -3,15 +3,16 @@
 //
 package io.deephaven.parquet.compress;
 
+import com.fasterxml.jackson.databind.util.ByteBufferBackedInputStream;
```

Comment:
Hehe; I recently added `io.deephaven.json.jackson.ByteBufferInputStream`, and told Ryan I would make it more general when somebody needed to use it. It's not appropriate to depend on jackson json databind for this, we should make available our version.

Reply:
I think you meant `io.deephaven.io.streams.ByteBufferInputStream`. Cool, will use that.

Reply:
I just found another  `io.deephaven.io.streams.ByteBufferInputStream`. It looks similar and a bit more exhaustive than `io.deephaven.json.jackson.ByteBufferInputStream` since it also implements `DataInput`.
I can try using that too.